packages/core/src/__tests__/dispatcher.test.ts

```diff
@@ -1,107 +1,127 @@
-import { Dispatcher }  from '../dispatcher'
-import CID from 'cids'
-import { Document } from "../document"
-import { TileDoctype } from "@ceramicnetwork/doctype-tile"
-import DocID from "@ceramicnetwork/docid";
-import { CommitType, DocState, DoctypeHandler, LoggerProvider } from '@ceramicnetwork/common';
+import { Dispatcher } from '../dispatcher';
+import CID from 'cids';
+import { Document } from '../document';
+import { TileDoctype } from '@ceramicnetwork/doctype-tile';
+import DocID from '@ceramicnetwork/docid';
+import {
+  AnchorService,
+  CeramicApi,
+  CommitType,
+  DocState,
+  DoctypeHandler,
+  LoggerProvider,
+} from '@ceramicnetwork/common';
 import { serialize, MsgType } from '../pubsub/pubsub-message';
 import { Repository } from '../state-management/repository';
 import { delay } from '../pubsub/__tests__/delay';
-import tmp from 'tmp-promise'
+import tmp from 'tmp-promise';
 import { LevelStateStore } from '../store/level-state-store';
 import { PinStore } from '../store/pin-store';
 import { RunningState } from '../state-management/running-state';
+import { FauxStateValidation, RealStateValidation } from '../state-management/state-validation';
+import { ContextfulHandler } from '../state-management/contextful-handler';
+import { TaskQueue } from '../pubsub/task-queue';
+import { ConflictResolution } from '../conflict-resolution';
 
-const TOPIC = '/ceramic'
-const FAKE_CID = new CID('bafybeig6xv5nwphfmvcnektpnojts33jqcuam7bmye2pb54adnrtccjlsu')
-const FAKE_CID2 = new CID('bafybeig6xv5nwphfmvcnektpnojts44jqcuam7bmye2pb54adnrtccjlsu')
-const FAKE_DOC_ID = DocID.fromString("kjzl6cwe1jw147dvq16zluojmraqvwdmbh61dx9e0c59i344lcrsgqfohexp60s")
+const TOPIC = '/ceramic';
+const FAKE_CID = new CID('bafybeig6xv5nwphfmvcnektpnojts33jqcuam7bmye2pb54adnrtccjlsu');
+const FAKE_CID2 = new CID('bafybeig6xv5nwphfmvcnektpnojts44jqcuam7bmye2pb54adnrtccjlsu');
+const FAKE_DOC_ID = DocID.fromString('kjzl6cwe1jw147dvq16zluojmraqvwdmbh61dx9e0c59i344lcrsgqfohexp60s');
 
 const ipfs = {
   pubsub: {
     ls: jest.fn(async () => Promise.resolve([])),
     subscribe: jest.fn(),
     unsubscribe: jest.fn(async () => Promise.resolve()),
-    publish: jest.fn()
+    publish: jest.fn(),
```

Comment:
I assume you have an editor that is auto-performing these small formatting changes?  Don't worry about it for this review, but for the future I would appreciate it if you could turn that off for now, as all these little changes create a lot of noise in the code reviews that I then have to sift through.  At some point we should enable an auto-formatter and do a single pass over the entire codebase to enforce a single style.  But until then, having these little style changes mixed in with actual functional changes is very distracting and slows down the review process

Reply:
makes sense. thanks.